bugmon_tc/monitor/tasks.py

```diff
@@ -103,41 +65,136 @@ def task(self):
                 "maxRunTime": MAX_RUNTIME,
             },
             "priority": "high",
-            "workerType": self.WORKER_TYPE,
+            "workerType": self.worker_type,
             "retries": 5,
             "routes": ["notify.email.jkratzer@mozilla.com.on-failed"],
             "schedulerId": "-",
             "scopes": self.scopes,
             "tags": {},
         }
 
+    @property
+    def scopes(self):
+        """Scopes applied to the task"""
+        return [
+            "docker-worker:capability:device:hostSharedMemory",
+            "docker-worker:capability:device:loopbackAudio",
+            "docker-worker:capability:privileged",
```

Comment:
Do you also need: `docker-worker:capability:disableSeccomp` ?

Reply:
Thank you.  I was going to reach out and ask why additional scopes I would need.